app/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Thanik Bhongbhibhat <email@thanik.me>",
   "license": "MIT",
   "scripts": {
-    "start": "parcel serve './src/index.html' --port 8000 --lazy",
+    "start": "parcel serve './src/index.html' --port 8000 --lazy --https",
```

Comment:
`https` mainly because I want `parcel` to serve hot reload via `wss` instead of `ws` so `ingress-envoy` proxies the upgrade request properly